src/index.html

```diff
@@ -9,10 +9,60 @@
     <title>Stars</title>
     <link
       rel="stylesheet"
-      href="./style.css"
+      href="./styles/modifiers.css"
+    />
+    <link
+      rel="stylesheet"
+      href="./styles/stars.css"
```

Comment:
Ensure that the `stars.css` file exists in the `styles` directory. This file is essential for the styling of the stars in your project.